Header header
float64 max_cluster_distance
int32 min_cluster_size
LaserCluster[] clusters
